js/client-mainmenu.js

```diff
@@ -1030,7 +1159,7 @@
 
 			var teams = Storage.teams;
 			if (!teams.length) {
-				return '<button class="select teamselect" name="team" disabled>You have no teams</button>';
```

Comment:
Why remove this?

Reply:
when there is no team on the storage the renderteams function always return disabled button

Reply:
That is intentional behavior - it shouldn't be enabled if there are no teams.